docs/finality-provider-operation.md

```diff
@@ -16,40 +14,45 @@ gain an overall understanding of the finality provider.
 
 ## Table of Contents
 
-1. [System Requirements](#1-system-requirements)
-2. [Keys Involved in Finality Provider Operation](#2-keys-involved-in-finality-provider-operation)
-3. [Install Finality Provider Toolset](#3-install-finality-provider-toolset)
-4. [Setting up the EOTS Daemon](#4-setting-up-the-eots-daemon)
-   1. [Initialize the EOTS Daemon](#41-initialize-the-eots-daemon)
-   2. [Add an EOTS Key](#42-add-an-eots-key)
-      1. [Create an EOTS key](#421-create-an-eots-key)
-      2. [Import an existing EOTS key](#422-import-an-existing-eots-key)
-   3. [Starting the EOTS Daemon](#43-starting-the-eots-daemon)
-       1. [Migration guide test to file keyring backend](#431-migration-guide-test-to-file-keyring-backend)
-       2. [Unlock file-based keyring](#432-unlock-file-based-keyring)
-5. [Setting up the Finality Provider](#5-setting-up-the-finality-provider)
-   1. [Initialize the Finality Provider Daemon](#51-initialize-the-finality-provider-daemon)
-   2. [Add key for the Babylon Genesis account](#52-add-key-for-the-babylon-genesis-account)
-   3. [Configure Your Finality Provider](#53-configure-your-finality-provider)
-   4. [Starting the Finality Provider Daemon](#54-starting-the-finality-provider-daemon)
-6. [Finality Provider Operation](#6-finality-provider-operations)
-   1. [Create Finality Provider](#61-create-finality-provider)
-   2. [Rewards](#62-rewards)
-   3. [Set Up Operation Key](#63-set-up-operation-key)
-   4. [Start Finality Provider](#64-start-finality-provider)
-   5. [Status of Finality Provider](#65-status-of-finality-provider)
-   6. [Edit finality provider](#66-edit-finality-provider)
-   7. [Jailing and Unjailing](#67-jailing-and-unjailing)
-   8. [Slashing](#68-slashing-and-anti-slashing)
-   9. [Prometheus Metrics](#69-prometheus-metrics)
-7. [Recovery and backup](#7-recovery-and-backup)
-   1. [Critical assets](#71-critical-assets)
-   2. [Backup recommendations](#72-backup-recommendations)
-   3. [Recover finality-provider db](#73-recover-finality-provider-db)
-      1. [Recover local status of a finality provider](#731-recover-local-status-of-a-finality-provider)
-      2. [Recover public randomness proof](#732-recover-public-randomness-proof)
-
-## 1. System Requirements
+1. [Prerequisites](#1-prerequisites)
+2. [System Requirements](#2-system-requirements)
```

Comment:
We should have system requirements for the EOTS daemon as well. @filippos47 @maiquanghiep thoughts?

Reply:
ok to add this in a separate PR and track it in an issue